src/python/library/tritonclient/utils/__init__.py

```diff
@@ -181,6 +184,9 @@ def triton_to_np_dtype(dtype):
         return np.float64
     elif dtype == "BYTES":
         return np.object_
+    # FIXME: Add proper bfloat16 support instead of fp16 WAR
+    elif dtype == "BF16":
+        return np.float16
```

Comment:
@jbkyang-nvi this conversion for triton<->np type is a temporary WAR to test identity backend with existing `L0_backend_identity`. Once proper python client support for bf16 is added, you should be able to tweak this and use the existing test.

Reply:
We shouldn't add dummy implementation to the client library...

Reply:
I'll see if the WAR can solely be done on test side without dummy implementation in client